docs/DeveloperGuide.md

```diff
@@ -219,10 +219,15 @@ It then gets the filepath of the `Tag` before safely deleting it. Then, a new `T
 changes the current directory of the HelloFile internal File Explorer. `CommandException` is thrown if the given directory 
 is invalid, cannot be found, or cannot be set as the current directory (*e.g. the given directory is not a folder*).
 
-CdCommand calls `setAddress` in `CurrentPath` to set the current directory to the absolute address parsed from the user input.
-Then, `CurrentPath` will update the list of children files `FileList` to fit the new current directory.
-Moreover, it also updates `javafx.scene.control.ListView` in `FileExplorer` as `ListView` is bound to the `FileList` of the children files 
-under the `CurrentPath`.
+This sequence diagram shows a successful execution of `CdCommand`.<br>
+![FindSequenceDiagram](images/CdCommandSuccessSequenceDiagram.png)
```

Comment:
I think you forget to change the FindSequenceDiagram.

Reply:
Oooo, ya, thanks for pointing that out

Reply:
Updated